debian/golang-go.install

```diff
@@ -1,5 +1,9 @@
+VERSION /usr/lib/go/
 bin/go /usr/lib/go/bin/
 bin/gofmt /usr/lib/go/bin/
+pkg/*_* /usr/lib/go/pkg/
+pkg/include /usr/lib/go/pkg/
```

Comment:
The contents of this directory are arch-independent so maybe this should be in /usr/share? and linked from /usr/lib?